### ISSUE TITLE
Rezept: Gesamtmenge pro Zutat on the fly

### ISSUE THREAD

**Issue by Rasinari** [kleiner-brauhelfer/kleiner-brauhelfer-2#262]
Es wäre wünschenswert wenn die benötigte Menge pro Zutat in Verbindung mit dem Ausstoß sofort on the fly angezeigt wird – und nicht wie jetzt die verfügbare Gesamtmenge.

Unter dem Eintrag „Vorhanden“ ist dafür Platz, z.B. als „Sudmenge“.
<img width="657" alt="Gesamtmenge" src="https://user-images.githubusercontent.com/51321250/194309948-c2695ac8-4dd9-4419-bf5c-ea25b251527b.png">

**Comment by BourgeoisLab** [comment]
Mir ist nicht klar, was du meinst. Du willst das Feld "Menge" unterhalb vom Feld "Vorhanden" haben?

**Comment by Rasinari** [comment]
Richtig. Ich bin schon mehrfach darüber gestolpert. Ein Direktvergleich ist besser, besonders dann, wenn man die Reproduktion des Bieres im Blick hat oder die Wirtschaftlichkeit, die Restmengen vermeidet.
